Orange/canvas/application/canvasmain.py

```diff
@@ -1057,6 +1061,23 @@ def set_new_scheme(self, new_scheme):
 
         old_scheme.deleteLater()
 
+    def ask_clear_report(self):
+        from Orange.canvas.report.owreport import OWReport
```

Comment:
You're right in general, but in this case it wouldn't work since the two modules would mutually import each other.